receiver/iisreceiver/metadata.yaml

```
@@ -0,0 +1,117 @@
+name: iisreceiver
+
+attributes:
+  direction:
+    value: direction
+    description: direction data is moving.
+    enum:
+      - sent
+      - received
+  request_type:
```

```suggestion
  request_type:
```
Typically attribute names do not contain underscores. I think either `request` or `type` is sufficient here anyways.

---

Updated